utils/DiscordBot.ts

```diff
@@ -0,0 +1,8 @@
+import * as Discord from 'discord.js'
+
+const DiscordBot = new Discord.Client()
+
+DiscordBot.on('ready', () => console.log('I\'m Ready'))
+DiscordBot.login(process.env.DISCORD_TOKEN)
+
+export default DiscordBot
```

Comment:
why need this?

Reply:
Cause its sexy

Reply:
DJS handles ratelimit and I need to fetch user infos from KOREANBOTS discord